django_feature_flip/helpers.py

```diff
@@ -19,4 +19,23 @@ def __getattr__(self, name):
         return getattr(self.instance, name)
 
     def enabled(self, feature_name, actor=None):
-        return Feature.objects.get(name=feature_name).enabled(actor)
+        feature = Feature.objects.get(name=feature_name)
+        if feature.enabled(actor):
+            return True
+
+        if actor:
+            return self.feature_enabled_for_a_group_of_actor(feature, actor)
+
+        return False
+
+    def register(self, group_name, func):
+        Group(name=group_name).save()
```

Comment:
Should we use [get_or_create](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#get-or-create) here? Otherwise if you try to register the same name more than once it will create duplicated groups.

```suggestion
        Group.objects.get_or_create(name=group_name)
```